src/utils.js

```diff
@@ -164,6 +164,11 @@ export const formatLiveLinks = (link) => {
 		link = link.replace('?path=/story/', '');
 	}
 
+	if(link.includes('http') || link.includes('https')){
+		link = link.match(/^(([^:\/\s]+):\/?\/?([^\/\s@]*@)?([^\/@:]*)?:?(\d+)?)?(\/[^?]*)?(\?([^#]*))?(#[\s\S]*)?$/)[6];
```

Comment:
This regex returns an array of values and the 6th value is always the path that's why its hardcoded